seungjong/app.js

```diff
@@ -1,34 +1,33 @@
 require('dotenv').config();
-
-const cors = require('cors');
-const logger = require('morgan');
+const cors = require("cors");
+const logger = require("morgan");
 const express = require('express');
-const { DataSource } = require('typeorm');
-
+const { DataSource } = require("typeorm");
 const app = express();
+const port = 4000;
```

Comment:
```suggestion
const PORT = process.env.PORT;
```
server port number는 하드코딩하는 것이 아닌, 환경변수의 값을 사용할 수 있도록 해주세요.